drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -849,7 +849,7 @@ void AudioDriverWASAPI::thread_func(void *p_udata) {
 
 					// fixme: Only works for floating point atm
 					for (UINT32 j = 0; j < num_frames_available; j++) {
-						int32_t l, r;
+						int32_t l = 0, r = 0;
```

Comment:
It's super rare in the codebase to initialize multiple variables on the same line. And they are single letter variables too. We could take this opportunity to rename them.

```suggestion
						int32_t left = 0;
						int32_t right = 0;
```